install.sh

```diff
@@ -532,7 +532,7 @@ EOABORT
     )"
   elif version_lt "${macos_version}" "10.11"
   then
-    abort "Your OS X version is too old."
+    abort "Your macOS version is too old."
```

Comment:
OS X was renamed to macOS in version 10.12, but this message shows only if the version is less than 10.11, so "OS X" here is probably correct.

Reply:
Yes, note that above it also says Mac OS X for versions below 10.7.